src/_i18n/en.yml

```diff
@@ -629,3 +629,13 @@ projects:
                      <p>Part of the feature set included a reporting tool to provide the business with detailed reports on the event. We spent time to help train the AWS marketing team on how to use the system.</p>
     outcome: <p>Given the tight feedback loops and real-time interactions with the Product Owner, we were able to deliver both the MVPs for the project and several additional features ahead of the deadline. The event team were able to respond quickly to demand via the real-time data analysis (for instance session scheduling, staff rotas) and report session take-up and popularity to stakeholders.</p>
              <p>AWS invited Codurance to add additional features to the system so that it could be used at future events. Our team also gave a talk on Software Craftsmanship at the AWS Summit London 2016.</p>
+  MANDS:
+    link-text: Case study at M&S
+    short_description: We helped transition their processes to cater for new projects
+    long_description: We provided mentoring and support at M&S, while there was a cultural transformation helping them prepare for their new projects and environment.
+    client_description: Marks and Spencer is a major British multinational retailer, specialising in the selling of clothing, home products and luxury food products.
+    the_challenge: <p>Marks and Spencers’ engineering department was entering a period of cultural change, with people being re-assigned to new teams ahead of several large digital projects. 
+    our_sc_approach: <p>With our connections to the wider community and passion for Software Craftsmanship, Codurance were seen as a partner could help to raise both code quality and to help establish a new team with this department through mentoring and guidance.
```

Comment:
I think there is something missing in `Codurance were seen as a partner could help to raise both code quality `, that could help?